atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManager.java

```diff
@@ -83,7 +85,7 @@
     private static final int NUM_RETRIES = 10;
 
     final MetricsManager metricsManager;
-    final KeyValueService keyValueService;
```

Comment:
here and elsewhere, this needs to be renamed to `keyValueServiceManager`, as a KVS is still a concept that is different!

Reply:
Yes: I kept this as is on purpose to keep the diff smaller. Mind if I FLUP with this?